src/lib/db-listeners/db-comments-listener.ts

```diff
@@ -0,0 +1,53 @@
+import { DB } from "../db";
+import { TileCommentModel, TileCommentsModelType, TileCommentsModel } from "../../models/tools/tile-comments";
+import { forEach } from "lodash";
+
+export class DBCommentsListener {
+  private db: DB;
+  private commentsRef: firebase.database.Reference | null = null;
+
+  constructor(db: DB) {
+    this.db = db;
+  }
+
+  public start() {
+    this.commentsRef = this.db.firebase.ref(
+      this.db.firebase.getUserDocumentCommentsPath(this.db.stores.user)
+    );
+    this.commentsRef.on("child_changed", this.handleUpdateComments);
+    this.commentsRef.on("child_added", this.handleUpdateComments);
+  }
+
+  public stop() {
+    if (this.commentsRef) {
+      this.commentsRef.on("child_changed", this.handleUpdateComments);
```

Comment:
```
on => off
```
found while reviewing Matt's related PR

Reply:
I'm going to patch both of these on my branch.

Reply:
On the teacher curation branch, I fixed this in both `db-comments-listener.ts` and `db-stars-listener.ts`.